### PR TITLE
Basic end to end tests for crypto module

### PR DESCRIPTION
**Pull request by andreyshtamburg**
- Introduced tests directory
- Implemented basic end to end test for crypto module

### CODE REVIEW

File: eve/tests/e2e/test_crypto.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from eve.config import DEFAULT_COIN, CURRENCY
+
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+
+def test_price_returned_with_default_options(runner):
+    result = runner.invoke(cli, ["price"])
```
Comment (IVIURRAY) [inline eve/tests/e2e/test_crypto.py:14]: I presume this actually calls the underlying API? - So we're doing a full integration test with the API, compared to a unit test to test just functionality that should not make any network requests.

**Comment by IVIURRAY** [comment]
Looks pretty good to me - would be cool to get these to run on each merge request

File: eve/tests/e2e/test_crypto.py (hunk 1 above)
Comment (andreyshtamburg) [reply eve/tests/e2e/test_crypto.py:14]: Yes, these are an actual calls to the API.
I was thinking that here we might have 2 types of tests. End to end where we cover big green path scenarios(that is what this PR for) and component/unit where we can mock stuff and cover more)
However, I'm not sure how exactly should I organise neither end to end nor unit/component tests

**Comment by andreyshtamburg** [comment]
> Looks pretty good to me - would be cool to get these to run on each merge request

Already prepared a branch for that.
I'd also suggest to run [black](https://black.readthedocs.io/en/stable/) code formatter validation.
What do you think?

**Comment by IVIURRAY** [comment]
> black

Ohh yeah I love `black` - I normally do this with pre-commit hooks and file watchers inside pycharm but I would be interested to see how you do it!

File: eve/tests/e2e/test_crypto.py (hunk 1 above)
Comment (IVIURRAY) [reply eve/tests/e2e/test_crypto.py:14]: Yeah, I think unit tests can live under `tests/services/` and `e2e` tests like you've here.

**Comment by IVIURRAY** [comment]
I also ran this locally with the default `config.py` and they all failed. We probably want to add a `test_config.py` or make use of pytest's monkeypatch to set your env vars!!

**Comment by IVIURRAY** [comment]
I also found [this](https://click.palletsprojects.com/en/7.x/options/#multiple-values-from-environment-values) in the click documentation. So we can get the env var in the options of the command.

**Pull request merged**